src/core/manager.h

```diff
@@ -511,6 +511,9 @@ struct Manager {
         int executor_fd;
 
         unsigned soft_reboots_count;
+
+        /* Ambient capability set for use with *some* forked processes */
+        uint64_t starting_ambient_set;
```

Comment:
maybe say "original" rather than "starting"?

because there's a lot of "starting" going on. the service manager itself gets started (whcih i guess is meant here), but we also start services and so on. by using the wording "original" I think things are clearer, to emphasize that this is the original ambient set when we first saw things.

Reply:
DONE